src/components/TransformtoEcommerce.js

```diff
@@ -0,0 +1,128 @@
+import { Button } from "@newfold/ui-component-library";
+import { __ } from "@wordpress/i18n";
+import { ReactComponent as CheckMarkIcon } from "../icons/check.svg";
+import { ReactComponent as LockIcon } from "../icons/lockicon.svg";
+import { ReactComponent as TransformStore } from "../icons/transform-store.svg";
+import { ReactComponent as WonderCartIcon } from "../icons/wondercart.svg";
+import { ReactComponent as YithPluginsIcon } from "../icons/yithplugins.svg";
+import { NewfoldRuntime } from "../sdk/NewfoldRuntime";
+
+
+
+export function TransformtoEcommerce() {    
+    return (
+        <div className={"nfd-app-section-container nfd-bg-white nfd-w-full nfd-rounded-lg " + (!(NewfoldRuntime.isWoo) ? 'nfd--mt-12' : '')} id="nonecommerce-features-wrapper">        
+            <div className="nfd-app-section-content nfd-border nfd-rounded-lg nfd-px-8 nfd-pt-0 nfd-pb-8 nfd--mt-2 nfd-items-start" style={{ borderColor: "#E2E8F0" }}>
+                <div className="nfd-flex nfd-flex-wrap nfd-gap-x-10">
+                    <div className="nfd-flex-none nfd-mt-8 nfd-w-40">
+                        <TransformStore />
+                    </div>
+                    <div class="nfd-flex-1 nfd-mt-5 nfd-pt-0.5">
+                        <h1 className="nfd-text-2xl nfd-leading-9 nfd-font-medium" style={{ color: "#111729" }}>
+                            {__("Transform your store!", "wp-module-ecommerce")}
+                        </h1>
+                        <p style={{ fontSize: "0.813rem", color: "#4A5567" }} className="nfd-leading-5 nfd-w-5/6">
+                            {__("Give your online store a boost with our all-in-one eCommerce bundle filled with exclusive plugins and tools designed to help you sell your products and services like a pro.", "wp-module-ecommerce")}
+                        </p>
+                    </div>
+                    <div className="nfd-flex-none nfd-mt-8 nfd-self-end">
+
+
+                        <Button
+                            as="a"
+                            className="nfd-button nfd-button--upsell nfd-w-36"
+                            variant="upsell"
+                            data-ctb-id={"f95ccf1e-3028-4ea7-b2c2-847969348e8b"}   
+                            target="_blank"
+                            rel="noopener"
+                            id="buy-ecommerce-addon-btn"
+                        >
+                            <LockIcon className="nfd-w-5 nfd-h-5 nfd--ml-1 nfd-shrink-0" />
+                            {__("Buy Now", "wp-module-ecommerce")}
```

Comment:
what happens when the plugin is installed but not active, which you're showing to buy now,

Reply:
This button is to buy Ecommerce `addon` and not a plugin. So install/activate stuff does not apply here. Customer buys Ecommerce plan and that itself is activates Ecommerce feature for them.